### PR TITLE
Uweb 7221/expand comparison table

### PR DESCRIPTION
**Pull request by ultienne**

### CODE REVIEW

File: src/molecules/table/TableRow.ts [hunk 1]
```diff
@@ -2,7 +2,8 @@ import { Tooltip } from '../../atoms/tooltip/Tooltip';
 import { ComparisonTableCell } from './ComparisonTableCell';
 
 export class TableRow {
-    label!: string;
+    content!: string;
```
Comment (dan-entous) [inline src/molecules/table/TableRow.ts]: why do we need the `!` here?

Comment (ultienne) [reply src/molecules/table/TableRow.ts]: > `!` is a kind of type assertion. It's used to assert that even though the property doesn't have an initializer, we're sure it'll contain a value of the specified type in runtime, not undefined.

Comment (dan-entous) [reply src/molecules/table/TableRow.ts]: i’m wondering if it should be used in the model-definition. we haven’t been consistent with using it in this project. also looked it up (here)[https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator] and it seems that its intended use is in runtime code, not definition code. based on that i don't think we should include it in definitions, but in runtime code. what do you think?

Comment (ultienne) [reply src/molecules/table/TableRow.ts]: I removed the exclamation mark

**Pull request merged**